services/graph/pkg/config/config.go

```diff
@@ -45,51 +45,51 @@ type Spaces struct {
+	GroupNameAttribute string `yaml:"group_name_attribute" env:"OCIS_LDAP_GROUP_SCHEMA_GROUPNAME;LDAP_GROUP_SCHEMA_GROUPNAME;GRAPH_LDAP_GROUP_NAME_ATTRIBUTE" desc:"LDAP Attribute to use for the name of groups." deprecationVersion:"3.0" removalVersion:"3.1" deprecationInfo:"LDAP_GROUP_SCHEMA_GROUPNAME changing name for consistency" deprecationReplacement:"OCIS_LDAP_GROUP_SCHEMA_GROUPNAME"`
+	GroupIDAttribute   string `yaml:"group_id_attribute" env:"OCIS_LDAP_GROUP_SCHEMA_ID;LDAP_GROUP_SCHEMA_ID;GRAPH_LDAP_GROUP_ID_ATTRIBUTE" desc:"LDAP Attribute to use as the unique id for groups. This should be a stable globally unique ID like a UUID." deprecationVersion:"3.0" removalVersion:"3.1" deprecationInfo:"LDAP_GROUP_SCHEMA_ID changing name for consistency" deprecationReplacement:"OCIS_LDAP_GROUP_SCHEMA_ID"`
 
-	EducationResourcesEnabled bool `yaml:"education_resources_enabled" env:"LDAP_EDUCATION_RESOURCES_ENABLED;GRAPH_LDAP_EDUCATION_RESOURCES_ENABLED" desc:"Enable LDAP support for managing education related resources"`
+	EducationResourcesEnabled bool `yaml:"education_resources_enabled" env:"OCIS_LDAP_EDUCATION_RESOURCES_ENABLED;LDAP_EDUCATION_RESOURCES_ENABLED;GRAPH_LDAP_EDUCATION_RESOURCES_ENABLED" desc:"Enable LDAP support for managing education related resources" deprecationVersion:"3.0" removalVersion:"3.1" deprecationInfo:"LDAP_EDUCATION_RESOURCES_ENABLED changing name for consistency" deprecationReplacement:"OCIS_LDAP_EDUCATION_RESOURCES_ENABLED"`
 	EducationConfig           LDAPEducationConfig
 }
 
 // LDAPEducationConfig represents the LDAP configuration for education related resources
 type LDAPEducationConfig struct {
-	SchoolBaseDN      string `yaml:"school_base_dn" env:"LDAP_SCHOOL_BASE_DN;GRAPH_LDAP_SCHOOL_BASE_DN" desc:"Search base DN for looking up LDAP schools."`
-	SchoolSearchScope string `yaml:"school_search_scope" env:"LDAP_SCHOOL_SCOPE;GRAPH_LDAP_SCHOOL_SEARCH_SCOPE" desc:"LDAP search scope to use when looking up schools. Supported scopes are 'base', 'one' and 'sub'."`
+	SchoolBaseDN      string `yaml:"school_base_dn" env:"OCIS_LDAP_SCHOOL_BASE_DN;LDAP_SCHOOL_BASE_DN;GRAPH_LDAP_SCHOOL_BASE_DN" desc:"Search base DN for looking up LDAP schools." deprecationVersion:"3.0" removalVersion:"3.1" deprecationInfo:"LDAP_SCHOOL_BASE_DN changing name for consistency" deprecationReplacement:"OCIS_LDAP_SCHOOL_BASE_DN"`
+	SchoolSearchScope string `yaml:"school_search_scope" env:"OCIS_LDAP_SCHOOL_SCOPE;LDAP_SCHOOL_SCOPE;GRAPH_LDAP_SCHOOL_SEARCH_SCOPE" desc:"LDAP search scope to use when looking up schools. Supported scopes are 'base', 'one' and 'sub'." deprecationVersion:"3.0" removalVersion:"3.1" deprecationInfo:"LDAP_SCHOOL_SCOPE changing name for consistency" deprecationReplacement:"OCIS_LDAP_SCHOOL_SCOPE"`
```

Comment:
```suggestion
	SchoolBaseDN      string `yaml:"school_base_dn" env:"GRAPH_LDAP_SCHOOL_BASE_DN" desc:"Search base DN for looking up LDAP schools."`
	SchoolSearchScope string `yaml:"school_search_scope" env:"GRAPH_LDAP_SCHOOL_SEARCH_SCOPE" desc:"LDAP search scope to use when looking up schools. Supported scopes are 'base', 'one' and 'sub'."`
```
The `LDAP_` global variant is not necessary, the same for `OCIS_LDAP_` because this envvar is only used in the GRAPH sevice. No deprecation necessary as this is very specific and introduced not long time ago and the yaml keys stay the same.